modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/objectFields.js

```diff
@@ -106,7 +106,8 @@ export function getSelectedValue(value) {
 }
 
 export function getObjectFieldName({settingsContext}) {
-	const getAdvancedColumn = ({title}) => title.toLowerCase() === 'advanced';
+	const getAdvancedColumn = ({title}) =>
+		title.toLowerCase() === Liferay.Language.get('advanced').toLowerCase();
 	const fieldsFromAdvancedColumn = settingsContext.pages.find(
 		getAdvancedColumn
 	)?.rows[0].columns[0].fields;
```

Comment:
Hi @jgarcialr,

I think we could simplify this code by doing something like:

```
const fieldsFromAdvancedColumn = settingsContext.pages.find(
	({title}) => title === Liferay.Language.get('advanced')
)?.rows[0].columns[0].fields;
```

Reply:
Hi @carolmariaabb,

I'll add this change, thanks for the review :smile: